training/src/anemoi/training/config/training/default.yaml

```diff
@@ -33,8 +33,19 @@ swa:
   enabled: False
   lr: 1.e-4
 
-# use ZeroRedundancyOptimizer ; saves memory for larger models
-zero_optimizer: False
+# Optimizer settings
+optimizer:
+  zero: False # use ZeroRedundancyOptimizer ; saves memory for larger models
+  kwargs:
+    betas: [0.9, 0.95]
+
+# select forecaster
+forecaster: anemoi.training.train.forecaster.GraphForecaster
+
+# select strategy
+strategy:
```

Comment:
In the init of the strategy we include the `read_group_size` why don't specify this at config level?